tests/cupy_tests/creation_tests/test_basic.py

```diff
@@ -18,6 +18,14 @@ def test_empty(self, xp, dtype):
         a.fill(0)
         return a
 
+    @testing.slow
```

Comment:
It seems that `@testing.slow` is not defined in `cupy.testing`. Probably we need to define the decorator in `cupy/testing/attr.py`.